hydromt/workflows/forcing.py

```diff
@@ -414,6 +477,153 @@ def pet_makkink(temp, press, k_in, timestep=86400, cp=1005.0):
     return pet
 
 
+def penman_monteith(
+    temp,
+    temp_min,
+    temp_max,
+    press,
+    kin,
+    wind,
+    var_for_avp,
+    elevtn,
+    doy,
+    lat_rad,
+    var_for_avp_name,
+):
+    """
+    todo
+    """
+    # saturation vapor pressure svp [kPa] from temp [degC]
+    svp = pyeto.svp_from_t(temp)
+
+    # actual vapor pressure avp [kPa] from dewpoint temperature [degC]
+    if var_for_avp_name == "temp_dew":
+        avp = pyeto.avp_from_tdew(var_for_avp)
+    # actual vapor pressure avp [kPa] from relative humidity rh [%]
+    elif var_for_avp_name == "rh":
+        svp_tmin = pyeto.svp_from_t(temp_min)
+        svp_tmax = pyeto.svp_from_t(temp_max)
+        avp = pyeto.avp_from_rhmean(svp_tmin, svp_tmax, var_for_avp)
+    else:
+        variables_avp = ["temp_dew", "rh"]
+        ValueError(
+            f"Unknown method to calculate avp, select from variables {variables_avp}"
+        )
+
+    # slope of the sat vapor pressure curve
+    delta_svp = pyeto.delta_svp(temp)
+
+    # psychrometric constant
+    # press already calculated from elevation (or if available directly)
+    psy = pyeto.psy_const(press / 10)  # in this formula press should be in kPa
+
+    # ned rad
+    # first calc extraterrestrial rad (et_rad) and clear sky radiation (cs_rad)
+    sol_dec = pyeto.sol_dec(doy)
+    sha = pyeto.sunset_hour_angle(lat_rad, sol_dec)
+    ird = pyeto.inv_rel_dist_earth_sun(doy)
+    et_rad = pyeto.et_rad(lat_rad, sol_dec, sha, ird)
+    cs_rad = pyeto.cs_rad(elevtn, et_rad)
+    # then longwave outgoing
+    long_out = pyeto.net_out_lw_rad(
+        pyeto.celsius2kelvin(temp_min),
+        pyeto.celsius2kelvin(temp_max),
+        kin * 86400 / 1e6,
+        cs_rad,
+        avp,
+    )  # in this formula kin should be in [MJ m-2 day-1]
+
+    # then net rad
+    # first net rad short
+    net_rad_s = pyeto.net_in_sol_rad(
+        kin * 86400 / 1e6, albedo=0.23
+    )  # in this formula kin should be in [MJ m-2 day-1]
+    net_rad = pyeto.net_rad(net_rad_s, long_out)
+
+    # now eto....
+    pet = pyeto.fao56_penman_monteith(
+        net_rad, pyeto.celsius2kelvin(temp), wind, svp, avp, delta_svp, psy
```

Comment:
As discussed, maybe nicer to have the .celcius2kelvin function outside of the .fao56_penman_monteith function.

Reply:
agree, done in e3f5cc78d6a41ff9fd8a004b634d437671918440